docs/introduction/core-concepts/feature-hierarchy.md

```diff
@@ -2,36 +2,86 @@
 title: Feature Hierarchy
 sidebar_position: 1
 ---
-In DevCycle, Features, Variables, and Variations are key components that work together to manage and control different aspects of your application.
+In a traditional feature flagging platform, there is only type of entity to worry about: a `feature flag`! In these 
+platforms, a `feature flag` contains the targeting rules and possible values for its respective key, all rolled into one.
 
-## Features 
+DevCycle does things differently. Rather than restricting you to one `feature flag` per set of targeting rules, we allow
+multiple `flags` to be governed by a single set of rules. In our platform, a value that you retrieve in your code is
+referred to as a `Variable`. For example, this line of code obtains a `Variable` from DevCycle using the unique key `headerText`:
+```typescript
+const headerText = devCycleClient.variable('headerText', 'Welcome to My Website!')
+```
 
-Features are the main elements that you want to control or experiment with in your application. 
+In order for DevCycle to provide different values for this `Variable`, it must be associated with a `Feature`. A `Feature`
+serves as a logical grouping of `Variables`, and defines the sets of possible values these `Variables` can receive.
+It also contains the targeting rules that decide who should receive what value. The sets of possible values are called 
+`Variations`. 
 
-> They can be anything from a new UI element to a backend algorithm. 
+You can think of a `Feature` as a set of related `Variables` that are all part of a new product feature, or should be 
+grouped together for organization purposes. A concrete example of this is a `Feature` like "Website Redesign" which
+can contain many `Variables` controlling different aspects of the design, like "headerText", "headerColor", "buttonColor", etc.
```

Comment:
```suggestion
grouped together for organization purposes. A concrete example of this is a `Feature` like "Website Redesign" which
can contain many `Variables` controlling different aspects of the design, like "headerText", "headerColor", "buttonColor", etc.   At DevCycle, we often have `Features` that contain two `Variables` - one that controls the front-end, and another that controls the back-end. 
```
```suggestion
grouped together for organization purposes. A concrete example of this is a `Feature` like "Website Redesign" which
can contain many `Variables` controlling different aspects of the design, like "headerText", "headerColor", "buttonColor", etc.
```
```suggestion
grouped together for organization purposes. A concrete example of this is a `Feature` like "Website Redesign" which
can contain many `Variables` controlling different aspects of the design, like "headerText", "headerColor", "buttonColor", etc.
```

Reply:
I think @katemacfarlane's edits are great! What do you think @ajwootto?

Once you've decided what to do with each of them, I think this is good to merge!

Reply:
it seems maybe a bit overly specific to me. If the idea is to kind of explain how we ourselves benefit from it, maybe just saying we have "multiple variables controlling both the back-end and front-end"? I don't think we usually have just "two"